src/main/webapp/dev/js/common/const.js

```diff
@@ -16,6 +16,7 @@ const StatusType = {
 };
 StatusType.DEFAULT = StatusType.INFO;
 
+// Sorted in a logical order, which is pre-determined.
```

Comment:
This comment only indicates there is a certain order but does not explain what order is that?

Reply:
Will do right away.